roseau/load_flow/io/dgs.py

```diff
@@ -378,7 +395,7 @@ def _generate_loads(
 
         # Balanced or Unbalanced
         s = [s_phase / 3, s_phase / 3, s_phase / 3] if sa == 0 and sb == 0 and sc == 0 else [sa, sb, sc]
-        loads[load_id] = PowerLoad(id=load_id, phases="abcn", bus=buses[bus_id], powers=s)
+        loads[load_id] = PowerLoad(id=load_id, bus=buses[bus_id], powers=s)
```

Comment:
@Saelyos this change and the change above for the source are made because we don't allow floating neutrals yet and the DGS import fails if the bus has "abc" phases while the load is defined as "abcn". Is this correct or should we always keep "abcn" here and allow floating neutrals?

Reply:
For the voltage source, we can keep the same phases as the bus, but the voltages need to be adapted for the delta voltage source. 

For the load, I'm actually not sure what the loads are supposed to be in the dgs file. We have assumed for now that the loads are wye loads, but I'm not sure it's true : there is a `phtech` attribute that we don't take into account.